appserver/tests/cts_smoke/test_dir.properties

```diff
@@ -0,0 +1,8 @@
+servlet-api-servlet=servlet/api/javax_servlet
+servlet-api-servlet-http=servlet/api/javax_servlet_http
+servlet-compat=servlet/compat
```

Comment:
As discussed, a logic of failing test run in case some new folder is being added but properties is not updated, should be added.